### PR TITLE
Add async-graphql example

### PR DESCRIPTION
**Pull request by sunli829**

### CODE REVIEW

**Comment by robjtede** [comment]
Hey @sunli829, can you update this to actix-web v3 so we can look at merging it.

**Comment by sunli829** [comment]
> Hey @sunli829, can you update this to actix-web v3 and rebase on master so we can look at merging it.

Okay, I upgraded to actix-web 3.0, thank you for your reply.

**Comment by sunli829** [comment]
I have updated the code, please review. 😁 @robjtede

File: Cargo.toml [hunk 1]
```diff
@@ -52,4 +52,6 @@ members = [
    "websocket-chat",
    "websocket-chat-broker",
    "websocket-tcp-chat",
+   "docker_sample",
+   "async-graphql-demo"
```
Comment (JohnTitor) [inline Cargo.toml:56]: `docker_sample` here should be removed as duplicated and `async-graphql-demo` should be ordered alphabetically.

File: async-graphql-demo/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+async fn index_playground() -> Result<HttpResponse> {
+    Ok(HttpResponse::Ok()
+        .content_type("text/html; charset=utf-8")
+        .body(playground_source(
+            GraphQLPlaygroundConfig::new("/").subscription_endpoint("/"),
+        )))
+}
+
+#[actix_rt::main]
```
Comment (robjtede) [inline async-graphql-demo/src/main.rs:21]: This is the recommended way now and can remove the actix-rt dependency.

```suggestion
#[actix_web::main]
```

Comment (sunli829) [reply async-graphql-demo/src/main.rs:21]: Thank you! 🙂

**Comment by robjtede** [comment]
This needs a readme similar to other ones ([here's a good reference](https://github.com/actix/examples/blob/master/diesel/README.md)) just to quickly explain the example and show folks what endpoints they can call.

**Review by robjtede — APPROVED** [review]
LGTM, thanks :)

**Pull request merged**